addons/test_mail/tests/test_mail_push.py

```diff
@@ -140,6 +140,13 @@ def test_notify_by_push_channel(self, push_to_end_point):
         push_to_end_point.assert_not_called()
         push_to_end_point.reset_mock()
 
+        self.env["discuss.channel.member"].search([
```

Comment:
You dont have default values for mute as far as I know, hence why forcing False ? Those are test users, they have no mute value, do they ?

Reply:
the problem is here https://github.com/odoo/odoo/pull/160148/files/f98e0ff5369f7a704e85ebf905abbbfb22b467ad#diff-5f4881b5889644a830bca19abd06b0ee075f57acead3ba2953dd1890ed2f066cR133 is setting the mute so they have a value now.

Reply:
L133
```py
self.env['discuss.channel.member'].search([
            ('partner_id', 'in', (self.user_email.partner_id + self.user_inbox.partner_id).ids),
            ('channel_id', 'in', (chat_channel + channel_channel + group_channel).ids),
        ]).write({
            'mute_until_dt': now + timedelta(days=5)
        })
```